graphqlapi/src/main/java/io/stargate/graphql/web/resources/schemafirst/NamespaceResource.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.graphql.web.resources.schemafirst;
+
+import graphql.GraphQL;
+import graphql.GraphqlErrorException;
+import io.stargate.graphql.web.RequestToHeadersMapper;
+import io.stargate.graphql.web.models.GraphqlJsonBody;
+import io.stargate.graphql.web.resources.Authenticated;
+import io.stargate.graphql.web.resources.GraphqlResourceBase;
+import java.util.regex.Pattern;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.container.AsyncResponse;
+import javax.ws.rs.container.Suspended;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** Serves the GraphQL schema deployed on each namespace. */
+@Singleton
+@Path(ResourcePaths.NAMESPACES)
```

Comment:
Would it make sense to add swagger to the REST APIs in here? I can create an issue if we think it'd be beneficial

Reply:
Now that I look FilesResource is the only rest in here?

Reply:
I'm not sure if that adds a lot of value. These resources are a direct implementation of the standard way of serving GraphQL over HTTP, as specified [here|https://graphql.org/learn/serving-over-http/]. Users will typically use a client or framework that abstracts those details, so I don't think they'll look for Swagger docs.

Reply:
Sure but isn't FilesResource.java actual REST? Or am I misunderstanding and it's also graphql over http?

Reply:
I think it's supporting stuff, where you actually expose the schema and directives, thus I would agree with @olim7t and not add this to Swagger as this should be used by a GQL client..